datadog_checks_base/datadog_checks/base/utils/db/statement_metrics.py

```diff
@@ -0,0 +1,119 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class StatementMetrics:
```

Comment:
Note that this class will be used in both Postgres and MySQL integrations (as well as SQL Server, Oracle, etc. when we support those).